### PR TITLE
Stories rework: VaValue

### PR DESCRIPTION
**Pull request by xiongmao86**
Story rework of VaValue.
Part of #3676.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

**Comment by xiongmao86** [comment]
I am very sorry that I forget to remove reluctant import, the pr is ready now.

**Comment by asvae** [comment]
Hey. Thanks!

We probably don't need last 2 stories (leave only Defaul), as VaValue component doesn't introduce any DOM. 
@m0ksem what do you think?

For first story it would be nice to have interaction test (you can check bactop for examples).

**Comment by m0ksem** [comment]
> Hey. Thanks!
> 
> We probably don't need last 2 stories (leave only Defaul), as VaValue component doesn't introduce any DOM. @m0ksem what do you think?
> 
> For first story it would be nice to have interaction test (you can check bactop for examples).

I should've been moved two last stories to docs demo (or I actually did it). Since we moved to stories from demos, I think we can remove two last stories. 

We can add story of usage VaValue with reactive object, not just boolean.

```html
<VaValue :defaultValue="{ name: '', age: 0 }" #default="v">
	<VaInput v-model="v.name" label="name" />
	<VaInput v-model="v.age" label="name" />
</VaValue>
```

I'm not sure if reactive object will work, but we need a story about it.

**Comment by xiongmao86** [comment]
Thanks, @asvae, @m0ksem. I don't know how to check whether the reactive object case work. Is this commit ok?

**Review by m0ksem — APPROVED** [review]

File: packages/ui/src/components/va-value/VaValue.stories.ts [hunk 1]
```diff
@@ -1,12 +1,47 @@
...
+    userEvent.click(button)
+    await sleep()
+    expect(button.innerText).toEqual('clicked')
+  })
+}
+
+export const ReactiveObject = () => ({
+  components: { VaValue, VaInput },
+  template: `
+    <VaValue :defaultValue="{ name: '', age: 0 }" #default="v">
```
[the remark this answers is not in the document]
Comment (m0ksem) [reply packages/ui/src/components/va-value/VaValue.stories.ts:42]: Never mind, you should use v.value.name in VaValue component and is fixed. Pushed this changes.

**Pull request merged**